ethereum/core/src/main/java/org/hyperledger/besu/ethereum/forkid/ForkIdManager.java

```diff
@@ -119,7 +156,7 @@ public boolean peerCheck(final ForkId forkId) {
     if (!isHashKnown(forkId.getHash())) {
       return false;
     }
-    return chainHeadSupplier.getAsLong() < forkNext
+    return chainHeadSupplier.get().getNumber() < forkNext
```

Comment:
this is a more verbose version of the same thing?

Reply:
Another for https://github.com/hyperledger/besu/commit/8b5e3ed1092bdd2b58851ba123976268a41044bf cc @jframe